integration/spark-common-test/src/test/resources/partition_data.csv

```diff
@@ -0,0 +1,27 @@
+id,vin,logdate,phonenumber,country,area,salary
```

Comment:
Hi @chenliang613 this csv data is already existed for partition example and test case. It's simple and clear to understand the partition concept. this PR just added two columns.